src/box/memtx_space.c

```diff
@@ -301,13 +297,13 @@ memtx_space_replace_all_keys(struct space *space, struct tuple *old_tuple,
 	for (i++; i < space->index_count; i++) {
 		struct tuple *unused;
 		struct index *index = space->index[i];
-		if (index_replace(index, old_tuple, new_tuple,
+		if (index_replace(index, needs_txm, old_tuple, new_tuple,
```

Comment:
An unnamed boolean argument looks terrible from the code readability point of view. It looks like this is the only place where you can possibly pass `true`. I don't understand why you can't call `index_replace_with_txm` here directly.

Reply:
Good point, didn’t think of it: applied your suggestion.